internal/node/reverse_proxy/handler_test.go

```diff
@@ -0,0 +1,24 @@
+package reverse_proxy
+
+import (
+	"testing"
+
+	"github.com/eagraf/habitat-new/core/state/node"
+	"github.com/eagraf/habitat-new/internal/node/config"
+)
+
+func TestGetHandlerFromRule(t *testing.T) {
+
+	fakeConfig, err := config.NewNodeConfig()
+	if err != nil {
+		t.Error(err)
+	}
+
+	redirectRule := &node.ReverseProxyRule{
+		ID:     "redirect1",
+		Type:   node.ProxyRuleRedirect,
+		Target: "http://fake-target/api",
+	}
+
+	getHandlerFromRule(redirectRule, fakeConfig)
```

Comment:
we're not asserting on anything in this test? +1 no err handler

Reply:
Nice catch